src/index.css

```diff
@@ -2,8 +2,42 @@
 @tailwind components;
 @tailwind utilities;
 
+html {
+  background-color: #0d0030;
```

Comment:
Why do we have this `diff`, didn't I merge this changes into `main`?

Reply:
I don't know, but I took your branch as a fork